### PR TITLE
Generate review model

### PR DESCRIPTION
**Pull request by mxng**
## Why

No review model

## What

Added review model with validation

### CODE REVIEW

**Comment by mxng** [comment]
Please let me know if I miss anything

File: app/models/review.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :restaurant
+
+  validates :title, presence: true, length: { in: 3..30 }
+  validates :content, presence: true, length: { in: 5..300 }
+  validates :rating, presence: true
```
Comment (TeeWhyKay) [inline app/models/review.rb]: thumbs up/thumbs down is 0 : 1 respectively?

Comment (mxng) [reply app/models/review.rb]: Ohh yeah. Either that, or 👎 = -1 while 👍 = 1?

Comment (mxng) [reply app/models/review.rb]: What should I change this to? validate only 0 and 1?

Comment (TeeWhyKay) [reply app/models/review.rb]: 👎 -> 0, 👍 ->  1 

in the future the rating range can be 1-5 (whole integers)

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :restaurant
+
+  validates :title, presence: true, length: { in: 3..30 }
+  validates :content, presence: true, length: { in: 5..300 }
```
Comment (TeeWhyKay) [inline app/models/review.rb]: i feel like title and content should be optional to encourage users to review?

Comment (mxng) [reply app/models/review.rb]: Let's remove content and title then

**Review by TeeWhyKay — APPROVED** [review]
great job on the review model 🌟🌟🌟🌟🌟

**Pull request merged**